crates/storage/engines/libmdbx.rs

```diff
@@ -461,6 +460,15 @@ table!(
     ( Payloads ) u64 => BlockRLP
 );
 
+// Current filters
+table!(
```

Comment:
Why use an `libmdbx` table for this instead of simply an in-memory hash map? I don't think we care much about persisting filters

Reply:
Done!